contrib/babelfishpg_common/Version.config

```diff
@@ -1,4 +1,4 @@
 BBFPGCMN_MAJOR_VERSION=3
-BBFPGCMN_MINOR_VERSION=2
+BBFPGCMN_MINOR_VERSION=3
```

Comment:
is this change intended?

Reply:
Yes actually we had to include new sql changes for the spatial datatypes, so we had bump the version and create new upgrade file 'babelfishpg_common--3.2.0--3.3.0.sql', we have also automated this generation so that we could use seperate flag to enable tds types sql files